<ui version="4.0" >
 <class>ProfileDataHashlistWidgetUI</class>
 <widget class="QWidget" name="ProfileDataHashlistWidgetUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>92</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>400</width>
    <height>92</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <layout class="QHBoxLayout" name="_2" >
        <item>
         <widget class="QLabel" name="label_format" >
          <property name="minimumSize" >
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>Format</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KComboBox" name="kcombobox_format" />
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" name="_3" >
        <item>
         <widget class="QLabel" name="label_mask" >
          <property name="minimumSize" >
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>Name/Mask</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KLineEdit" name="klineedit_mask" />
        </item>
        <item>
         <widget class="KPushButton" name="kpushbutton_mask" >
          <property name="minimumSize" >
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>380</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
